RealmSwift-swift1.2/RealmConfiguration.swift

```diff
@@ -147,10 +147,12 @@ extension Realm {
 
         internal var rlmConfiguration: RLMRealmConfiguration {
             let configuration = RLMRealmConfiguration()
-            if self.path != nil {
+            if path != nil {
```

Comment:
For better or worse, that's what SwiftLint is for (even though it doesn't enforce this particular rule yet).